tests/backup/backup_kubevirt_test.go

```diff
@@ -2105,3 +2105,468 @@ var _ = Describe("{KubevirtVMRestoreWithAfterChangingVMConfig}", Label(TestCaseL
+				err = CreateRestoreWithValidation(ctx, restoreName, backupNames[i], make(map[string]string), make(map[string]string), DestinationClusterName, BackupOrgID, appContextsExpectedInBackup)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creation of restore [%s] from backup [%s]", restoreName, backupNames[i]))
+			}
+		})
+
+		// Default Restores with Replace Policy
+		Step(fmt.Sprintf("Default restore of backups by replacing the existing resources"), func() {
+			log.InfoD(fmt.Sprintf("Default restore of backups by replacing the existing resources"))
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			log.InfoD("Total backups to restore : %v", backupNames)
+			for i, bkpName := range backupNames {
+				restoreName = fmt.Sprintf("rreplace-%v-%s-%s", i, bkpName, RandomString(6))
+				log.InfoD("Restoring from the backup - [%s]", bkpName)
+				err = CreateRestoreWithReplacePolicy(restoreName, bkpName, make(map[string]string), DestinationClusterName, BackupOrgID, ctx, make(map[string]string), 2)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Default restore of backups by replacing the existing resources [%s]", restoreName))
+			}
+		})
+
+		Step(fmt.Sprintf("Default restore of backups by replacing to a new namespace"), func() {
+			log.InfoD(fmt.Sprintf("Default restore of backups by replacing to a new namespace"))
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			for i, bkpName := range backupNames {
+				restoreName = fmt.Sprintf("rreplace-ns-%v-%s-%s", i, bkpName, RandomString(6))
+				log.InfoD("Restoring from the backup - [%s]", bkpName)
+				actualBackupNamespaces, err := FetchNamespacesFromBackup(ctx, bkpName, BackupOrgID)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching namespaces from schedule backup %v - [%v]", actualBackupNamespaces, bkpName))
+				namespaceMapping := make(map[string]string)
+				for _, namespace := range actualBackupNamespaces {
+					if _, ok := namespaceMapping[namespace]; !ok {
+						namespaceMapping[namespace] = namespace + "-new"
+					}
+				}
+				log.InfoD("Backup namespace mapping : %v", namespaceMapping)
+				err = CreateRestoreWithReplacePolicyWithValidation(restoreName, bkpName, namespaceMapping, DestinationClusterName, BackupOrgID, ctx, make(map[string]string), 2, scheduledAppContexts)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Default restore of backups by replacing to a new namespace [%s]", restoreName))
+			}
+		})
+
+	})
+
+	JustAfterEach(func() {
+		defer EndPxBackupTorpedoTest(scheduledAppContexts)
+		defer func() {
+			log.InfoD("switching to default context")
+			err := SetClusterContext("")
+			log.FailOnError(err, "Failed to SetClusterContext to default cluster")
+		}()
+		ctx, err := backup.GetAdminCtxFromSecret()
+		log.FailOnError(err, "Fetching px-central-admin ctx")
+		opts := make(map[string]bool)
+		opts[SkipClusterScopedObjects] = true
```

Comment:
Might be unnecessary as we not destroying the apps.

Reply:
done